.github/workflows/deploy.yml

```diff
@@ -203,176 +203,109 @@ jobs:
-  #     - name: Set a Timestamp
-  #       id: timestampid
-  #       run: echo "timestamp=$(date --utc +%Y%m%d_%H%M%SZ)" >> "$GITHUB_OUTPUT"
-  #     - uses: actions/checkout@v4
-  #     - uses: actions/setup-node@v4
-  #       with:
-  #         node-version-file: ".nvmrc"
-  #     - name: Install dependencies
-  #       run: yarn install
-  #     # downloads artifact created from the test job
-  #     - name: Download reports from GitHub Actions Artifacts
-  #       uses: actions/download-artifact@v4
-  #       with:
-  #         name: playwright-html-report # download from previous job
-  #         path: downloaded-html-report # save as this when downloaded
-  #     - name: Push files to github pages
-  #       uses: peaceiris/actions-gh-pages@v4
-  #       with:
-  #         github_token: ${{ secrets.GITHUB_TOKEN }}
-  #         publish_dir: ./downloaded-html-report # publish downloaded dir to github pages
-  #         destination_dir: ${{ steps.timestampid.outputs.timestamp }}
-  #     # need to extract just org name for reassembling the github pages URL
-  #     - name: Extract Organization Name
-  #       id: extract-org
-  #       run: |
-  #         echo "ORG_NAME=$(echo $GITHUB_REPOSITORY | cut -d'/' -f1)" >> $GITHUB_ENV
-  #         echo "org name: ${ORG_NAME}"
-  #     # need to extract just the repo name for reassembling the github pages URL
-  #     - name: Extract Repository Name
-  #       id: extract-repo
-  #       run: |
-  #         echo "REPO_NAME=$(echo $GITHUB_REPOSITORY | cut -d'/' -f2)" >> $GITHUB_ENV
-  #         echo "repo name: ${REPO_NAME}"
-  #     # assembles org name, repo name, and unique timestamp to link to github pages url that was published
-  #     - name: Write URL in Summary
-  #       run: |
-  #         echo "## Playwright Test Results" >> $GITHUB_STEP_SUMMARY
-  #         echo "https://${ORG_NAME}.github.io/${REPO_NAME}/${{ steps.timestampid.outputs.timestamp }}/" >> $GITHUB_STEP_SUMMARY
+  upload-reports:
+    name: Upload Reports
+    needs:
+      - test
+    if: ${{ always() && github.ref_name != 'production' }}
+    runs-on: ubuntu-latest
+    outputs:
+      timestamp: ${{ steps.timestampid.outputs.timestamp }}
+    steps:
+      # create a unique folder name to put playwright reports in
+      - name: Set a Timestamp
+        id: timestampid
+        run: echo "timestamp=$(date --utc +%Y%m%d_%H%M%SZ)" >> "$GITHUB_OUTPUT"
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+      - name: Install dependencies
+        run: yarn install
+      # downloads artifact created from the test job
+      - name: Download reports from GitHub Actions Artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: playwright-html-report # download from previous job
+          path: downloaded-html-report # save as this when downloaded
+      - name: Push files to github pages
```

Comment:
Very cool!

Reply:
Agreed! Kudos to @dwhitestratiform for [setting this up in MFP](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/commit/92dd2ae6418cf72411f6e6f4fcf02c698da8b429) - I just copied his work 😎